### PR TITLE
Resolve conflicts for launch with model file after multiple tab support

### PR DESCRIPTION
**Pull request by ahmetcandiroglu**
#205

### CODE REVIEW

**Comment by ahmetcandiroglu** [comment]
Fixes #187 @kinimesi

**Comment by kinimesi** [comment]
@ahmetcandiroglu there are still a few problems:

 **Load from URL:** 
  - When the file extension is not `sbgn` (i.e. `xml` or `txt`), error is thrown to console.
  - When the URL is invalid (i.e. it does not exist), a new tab is created automatically
 
**Load with URI:**
  - When the URI is invalid, a new tab is created automatically
  - After the `Invalid URI` message is shown, when we click on `OK` `Paths By URI` menu is opened

**Comment by ahmetcandiroglu** [comment]
@kinimesi 

- I don't get errors when file extension is `txt` or `xml`.
- I thought new tab creation would be the default case if URL or URI query does not exist or they are invalid. What it should do instead?
- Invalid URI message use `PromptInvalidURI `component from `backbone-views`. I used it when you told me to use it earlier. Should I create a new component and not use `PromptInvalidURI` component or just not use it at all?

File: app/js/app-menu.js [hunk 1]
```diff
@@ -42,6 +45,101 @@ module.exports = function () {
...
+      loadFromURI(uri_path, chiseInstance);
+
+  }
+
+  function loadFromURL(filepath, chiseInstance){
+    var loadCallbackSBGNMLValidity = function (text) {
+      validateSBGNML(text);
+    }      
+    var loadCallbackInvalidityWarning  = function () {
+      new BackpromptInvalidFileView.render();
```
Comment (kinimesi) [inline app/js/app-menu.js]: @ahmetcandiroglu where is `BackpromptInvalidFileView` defined?

Comment (ahmetcandiroglu) [reply app/js/app-menu.js]: @kinimesi it should be `promptInvalidFileView.render()` and it is defined in `app-menu.js`.

**Comment by kinimesi** [comment]
@ahmetcandiroglu 
- if the URI or URL is invalid, just show the warning, there is no meaning in creating a new tab.
- when an invalid URI is entered, show appropriate message. When the user clicks on `OK` the warning should disappear (similar to invalid file warning message)

We (me and @metincansiper ) decided that most appropriate place of this code is `app-utilities.js` file. Please move your code from `app-menu.js` to `app-utilities.js`

**Comment by ahmetcandiroglu** [comment]
@kinimesi 

I have to change `PromptInvalidURIView.render()` function in order to just show a warning when an invalid URI is entered. Should I change it or create a new component inside `backbone-views`?

**Comment by kinimesi** [comment]
@ahmetcandiroglu create a new one. `PromptInvalidURIView` is already being used in somewhere else (modifying it will break other parts).

**Comment by ahmetcandiroglu** [comment]
@kinimesi 

requested changes have been implemented.

**Pull request merged**